system.h

```diff
@@ -126,5 +126,6 @@ extern int fdatasync(int fildes);
 #include "misc/fnmatch.h"
 
 #include <dlfcn.h>
+#include <errno.h>
```

Comment:
This doesn't belong in system.h. That beast is something to reduce until removed, rather than add to.

Reply:
FWIW, the dlfcn.h include doesn't belong there either.

Reply:
Also gone!